newIDE/app/src/ProjectManager/PublicGameProperties.js

```diff
@@ -125,6 +129,22 @@ function PublicGameProperties({
               </Trans>
             }
           />
+          {// This view is used for public game properties as well as project properties.
+          // This property is not shown in project properties.
+          setOwnerIds && (
```

Comment:
should the owners be before the authors? In order of importance?

Reply:
I though it was better to have owners after because they are picked from authors. Something like: "To which authors do you want to grant ownership?"